import-export-cli/integration/envSpecific_test.go

```diff
@@ -215,6 +215,285 @@ func TestEnvironmentSpecificParamsEndpointSecurityBasic(t *testing.T) {
 	testutils.ValidateEndpointSecurityDefinition(t, api, apiParams, importedAPI)
 }
 
+//  Import an API with the external params file that has HTTP/REST endpoints without load balancing or failover configs
```

Comment:
Let's change the descriptions to highlight the user who performs the test as well.
Example: Import an API with the external params file that has HTTP/REST endpoints without load balancing or failover configs as **super tenant user** with the **Internal/devops** role. 

The test names can be changed as well. (Since the names are lengthy, shall we go for a simpler name like below?)
Example: TestHttpRestEndpointParamsWithoutLoadBalancingOrFailoverSuperTenantDevops

@uvindra WDYT?